app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -340,6 +345,15 @@ export class DocumentSidePanel extends Component {
               />
             </TabContent>
             <TabContent for="toc" className="toc">
+              <div className="tocHeader">
+                <h1>
+                  <Translate>Table of contents </Translate>
+                </h1>
+                &nbsp;
+                <TocGeneratedLabel file={this.props.file}>
+                  <Translate>auto-created ⓘ </Translate>
```

Comment:
Probably keep the ⓘ outside of the translate?

Reply:
@konzz this allows not only to translate but to edit, in case you do not want the ⓘ, i think its good to have it there ?